README.md

```diff
@@ -1,59 +1,35 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+## My portfolio 
+
+This project is a mobile version of my portfolio. it willbe contious project that will show what am capable of doing a developer.
```

Comment:
Please, make these little changes below

- Capitalize `it`.
- Add some space between `willbe`
- spell `continous`
- It should be `what I am`, not `what am`